WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample/MainPage.xaml.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace NotificationHubSample
+{
+    [DesignTimeVisible(true)]
```

Comment:
Remove. It's not required. I know this is in the templates btw, rather than something you've added.